### PR TITLE
create blade template

### PR DESCRIPTION
**Pull request by tuanngoc1601**

### CODE REVIEW

**Comment by tuanngoc1601** [comment]
ready

File: resources/views/users/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                @method('PUT')
+                <div class="mt-4">
+                    <x-input-label for="name" :value="__('Name')"></x-input-label>
+
+                    <x-text-input id="name" class="block mt-1 w-full"
+                                  type="text"
+                                  name="name"
+                                  value="{{  $user->username  }}"
+                                  required autocomplete="name"></x-text-input>
+
```
Comment (ghost) [inline resources/views/users/edit.blade.php]: indent 4 spaces only

**Comment by tuanngoc1601** [comment]
ready

**Comment by ghost** [comment]
merged

**Pull request merged**